mpt_api_client/resources/commerce/assets.py

```diff
@@ -0,0 +1,100 @@
+from mpt_api_client.http import AsyncService, Service
+from mpt_api_client.http.mixins import (
+    AsyncCollectionMixin,
+    AsyncCreateMixin,
+    AsyncGetMixin,
+    AsyncUpdateMixin,
+    CollectionMixin,
+    CreateMixin,
+    GetMixin,
+    UpdateMixin,
+)
+from mpt_api_client.models import Model
+from mpt_api_client.models.model import ResourceData
+
+
+class Asset(Model):
+    """Asset resource."""
+
+
+class AssetTemplate(Model):
+    """Asset template resource."""
+
+
+class AssetsServiceConfig:
+    """Assets service config."""
+
+    _endpoint = "/public/v1/commerce/assets"
+    _model_class = Asset
+    _collection_key = "data"
+
+
+class AssetsService(
```

Comment:
I think we should use singular form for class names. I did this mistake in Adobe Sync code and realised it gives me a brain f... :-D . This is quite common opinion, but I forgot about it. I've seen this comment somewhere:

The tool to turn screws with is called "screw driver" not "screws driver".

:-D